pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -36,28 +37,28 @@ type connTrackSystem struct {
 	connTrack   NetFilterConnTrack
 }
 
-func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *connTrackSystem {
+func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) (*connTrackSystem, error, error) {
```

Comment:
it makes no sense to me that the function is returning 2 errors, that's hardly a good design IMO

and the needs of unit tests don't really justify that change

Reply:
Fixed. Declared a new function called setupConntrackParameters(), returns the first error encountered since setting nf_conntrack_acct and nf_conntrack_timestamp will succeed or fail at the same time.